packages/builders/scripts/testing/start-query-flows.js

```diff
@@ -7,7 +7,6 @@
 import { deeplyFulfilledObject, makeTracer } from '@agoric/internal';
 import { makeStorageNodeChild } from '@agoric/internal/src/lib-chainStorage.js';
 import { E } from '@endo/far';
-import { parseArgs } from 'node:util';
```

Comment:
I wonder how this worked before

Reply:
It never did - it was a part of a change introduced in this PR. I probably could've fixed up to avoid misleading here.